indico/modules/events/abstracts/controllers/abstract_list.py

```diff
@@ -107,12 +110,40 @@ def _process(self):
 
 class RHCreateAbstract(RHAbstractListBase):
     def _process(self):
-        abstract_form_class = make_abstract_form(self.event, session.user, notification_option=True, management=self.management)
+        is_invited = request.args.get('invited') == '1'
+        abstract_form_class = make_abstract_form(self.event, session.user, notification_option=True,
+                                                 management=self.management, invited=is_invited)
         form = abstract_form_class(event=self.event, management=self.management)
+        if is_invited:
+            del form.description
+            del form.submitted_contrib_type
+            del form.attachments
+            del form.send_notifications
+            del form.person_links
+
         if form.validate_on_submit():
+            if is_invited:
+                if form.data['users_with_no_account'] == 'existing':
+                    submitter = form.data.pop('submitter')
+                else:
+                    submitter = User(first_name=form.data.pop('first_name'), last_name=form.data.pop('last_name'),
+                                     email=form.data.pop('email'), is_pending=True)
+                    db.session.add(submitter)
+                    db.session.flush()
+
+                del form.users_with_no_account
+                del form.first_name
+                del form.last_name
+                del form.email
+                del form.submitter
+            else:
```

Comment:
i'd just assign `submitter = None` above the `if` and get rid of this `else` (a very small else after a somewhat big `if` doesn't improve code readability)